bitbots_quintic_walk/config/walking_wolfgang_optimization.yaml

```diff
@@ -8,10 +19,10 @@ walking:
 
   node:
     # update frequency of the engine
-    engine_freq: 1000.0
```

Comment:
Yes it is. Since we can now actually control the servos with 1kHz, it makes sense to run the walking at same speed for better performance (smoother trajectories, faster reaction times, ...). CPU wise it was not a problem, I tested it.

Reply:
But the change is from 1000 to 240 hz.

Reply:
true, thanks. I fixed it